### PR TITLE
Added VTCx places in Thailand

### PR DESCRIPTION
**Pull request by pepa65**
Hope this contribution is OK as is. I'm happy to do more if I find time, but these have been long overdue.

### CODE REVIEW

**Comment by pepa65** [comment]
Have added all locations in Thailand from `sao.cty` now.

**Comment by raveit65** [comment]
Are all new location tested?
And does all new locations give us weather infos?

I don't think we should locations without weather infos.

**Comment by pepa65** [comment]
I was assuming everything in sao.cty would therefore give weather info. I don't have the infrastructure to test it. I tried building, but got stuck. (See issue #86)

How about I change the pull request to only include the locations you have been able to confirm?

**Comment by pepa65** [comment]
Wait, I checked the locations in the current applet!

**Comment by raveit65** [comment]
I tested your first list again
```
Chiang Rai     no
Chiang Rai International Airport   yes
Mae Sariang    no
Buriram        yes
Chaiyaphum     no
Chanthaburi    no
Chumphon       yes
Kanchanaburi   no
Koh Samui      yes
Krabi          yes
Loei           yes
Mae Sariang    no
Mae Sot        yes
Mukdahan       no
Nakhon Phanom  yes
Nakhon Ratchasima  yes
Nakhon Sawan   no
Nakhon Si Thammarat yes
Narathiwat     yes
Nong Khai      no
Pattani        no
Phetchabun     yes
Phitsanulok    yes
Phetchaburi    no
Prachinburi    no
Prachuap Khirikhan no
Roi Et         yes
Sakon Nakhon   no
Satun          no
Sakon Nakhon   no
Songkhla       no
Sukhothai      yes
Trat           no
Uttaradit      no

Not selectable in preferences!
+        <!-- A city in Thailand -->
+        <name>Tak</name>
+        <coordinates>16.876689 99.138131</coordinates>
+        <location>
+          <name>Tak</name>
+          <code>VTPT</code>
+          <coordinates>16.876689 99.138131</coordinates>
+        </location>
```
I am still wondering that you open a PR, but you're not able to rebuild libmateweather with your changes.
And that you think that other people should spend their spare time to do your work.
Donation welcome.
https://mate-desktop.org/donate/

**Comment by pepa65** [comment]
I don't have these lines in HEAD (52c2249a7b3613e1860e34adecdf1c2540c4b475):
```
+        <!-- A city in Thailand -->
+        <name>Tak</name>
+        <coordinates>16.876689 99.138131</coordinates>
+        <location>
+          <name>Tak</name>
+          <code>VTPT</code>
+          <coordinates>16.876689 99.138131</coordinates>
+        </location>
```
This PR is basically an xml source file, I've checked its schema validity, but indeed, I haven't been able yet to build the library... Tried, but got stuck, I might need some help. But I know others are able to build. This contribution is more on the content of the weather stations' locations. Sorry, I would love to be able to build it. I thought that this kind of non-technical contributions could be useful for the project, because apparently nobody that has been involved has supplied this information. I would be grateful if someone else could build this.

File: data/Locations.xml.in [hunk 1]
```diff
@@ -6157,6 +6432,16 @@
           <coordinates>9.116667 99.350000</coordinates>
         </location>
       </city>
+      <city>
+        <!-- A city in Thailand -->
+        <name>Tak</name>
+        <coordinates>16.876689 99.138131</coordinates>
+        <location>
+          <name>Tak</name>
```
Comment (raveit65) [inline data/Locations.xml.in]: Really, this is from you!!!
```
<!-- A city in Thailand -->
        <name>Tak</name>
        <coordinates>16.876689 99.138131</coordinates>
        <location>
```

**Comment by raveit65** [comment]
Is current state of PR what you want to get merged or is something missing?
Otherwise i review it again....

**Comment by pepa65** [comment]
I have pushed the modified file every time after I received feedback from you, so the current state should automatically be updated every time (there is no longer an entry for "Tak").

**Comment by raveit65** [comment]
I fixed more problems with https://github.com/mate-desktop/libmateweather/commit/67521e5c2abc80e81c28052acc0d540a6fa7ce67
And cherry-picked to 1.24.

**Pull request closed without merging**